### PR TITLE
[ZEPPELIN-1287][branch-0.6] No need to call print to display output in PythonInterpreter

### PR DESCRIPTION
**Pull request by minahlee**
### What is this PR for?
Implement #1278 to merge branch-0.6

### What type of PR is it?
Bug Fix

### What is the Jira issue?
[ZEPPELIN-1287](https://issues.apache.org/jira/browse/ZEPPELIN-1287)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by minahlee** [comment]
\cc @zjffdu for review

**Comment by bzz** [comment]
@minahlee I think we also need to set appropriate "fix for" versions in JIRA before merging #1278  otherwise it will be very hard for you, as release manager to figure out changelog and attributions may get lost.

I have updated current one to both `0.7.0` for master and `0.6.1` for branch-0.6

LGTM

**Comment by minahlee** [comment]
@bzz Thanks for taking care of JIRA issue!

File: python/src/main/java/org/apache/zeppelin/python/PythonInterpreter.java [hunk 1]
```diff
@@ -138,8 +138,12 @@ public InterpreterResult interpret(String cmd, InterpreterContext contextInterpr
...
     String output = sendCommandToPython(cmd);
-    return new InterpreterResult(Code.SUCCESS, output.replaceAll(">>>", "")
-        .replaceAll("\\.\\.\\.", "").trim());
+
+    InterpreterResult result;
+    // TODO(zjffdu), we should not do string replacement operation in the result, as it is
+    // possible that the output contains the kind of pattern itself, e.g. print("...")
+    result = new InterpreterResult(Code.SUCCESS, output.replaceAll("\\.\\.\\.", ""));
+    return result;
   }
```
Comment (zjffdu) [inline python/src/main/java/org/apache/zeppelin/python/PythonInterpreter.java]: return it directly ?
```
return new InterpreterResult(Code.SUCCESS, output.replaceAll("\\.\\.\\.", ""));
```

**Comment by zjffdu** [comment]
Just one minor code style issue, rest LGTM, Thanks @minahlee for taking over this.

**Comment by minahlee** [comment]
@zjffdu Thanks for review!
Merge if there is no more discussion

**Pull request closed without merging**